### PR TITLE
1.6.5 release notes on master

### PR DESCRIPTION
**Pull request by George-Aeillo**

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -3,6 +3,20 @@ title: Release Notes
 weight: 2
 draft: false
 ---
+## v1.6.5
+#### Enhancements:
+- Reduced nginx-ingress controller image size.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:8]: -> NGINX Ingress Controller

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -3,6 +3,20 @@ title: Release Notes
...
+- Reduced Fluentd image size.
+- Cluster Class Support expanded.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:10]: -> Cluster class support (not sure if class should be capitalized; please find out who gave you this enhancement statement if you let me know, then I'll ask them about it.)

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -3,6 +3,20 @@ title: Release Notes
...
+- Uptake Rancher 2.7.5 Dashboard.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:11]: -> Uptake??? Do you mean Updated Rancher 2.7.5 dashboard (lowercase)

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -3,6 +3,20 @@ title: Release Notes
...
+- Published publicly available VPO Helm Charts.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:12]: -> VPO Helm charts.

File: content/en/docs/releasenotes/_index.md [hunk 5]
```diff
@@ -3,6 +3,20 @@ title: Release Notes
...
+
+#### Fixes:
+- Fixed bug to tolerate the absence of the VMC caSecret field.
+- Fixed issue where authproxy wouldn't automatically pick up a new configuration.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:16]: -> Fixed an issue

File: content/en/docs/releasenotes/_index.md [hunk 6]
```diff
@@ -3,6 +3,20 @@ title: Release Notes
...
+## v1.6.5
+#### Enhancements:
+- Reduced nginx-ingress controller image size.
+- Reduced Fluentd image size.
+- Cluster Class Support expanded.
+- Uptake Rancher 2.7.5 Dashboard.
+- Published publicly available VPO Helm Charts.
+
+#### Fixes:
+- Fixed bug to tolerate the absence of the VMC caSecret field.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:15]: -> Fixed an issue 
-> to tolerate the absence of the VMC caSecret field. (??? Please explain to me what this means!)
-> `caSecret` (code font)

File: content/en/docs/releasenotes/_index.md [hunk 7]
```diff
@@ -3,6 +3,20 @@ title: Release Notes
...
+- Fixed issue where authproxy wouldn't automatically pick up a new configuration.
+- Fixed bug where Verrazzano CR wouldn't allow for empty volume fields on update.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:17]: -> Fixed an issue

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:8]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 2 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:10]: Ok! I have asked

File: content/en/docs/releasenotes/_index.md (hunk 4 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:12]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 5 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:16]: Thanks!

File: content/en/docs/releasenotes/_index.md (hunk 7 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md:17]: Thanks!

File: content/en/docs/releasenotes/_index.md [hunk 8]
```diff
@@ -3,6 +3,20 @@ title: Release Notes
 weight: 2
 draft: false
 ---
+## v1.6.5
+#### Enhancements:
+- Reduced NGINX Ingress controller image size.
+- Reduced Fluentd image size.
+- ClusterClass Support expanded.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:10]: -> ClusterClass support

**Review by rosemarymarano — APPROVED** [review]
Just one minor edit, otherwise fine.

File: content/en/docs/releasenotes/_index.md (hunk 8 above)
Comment (George-Aeillo) [reply content/en/docs/releasenotes/_index.md]: Thanks!

**Pull request merged**